setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='VacasaConnect',
-    version='0.7.1',
+    version='0.6.3',
```

Comment:
It's odd I had to drop this down. But I there was a 0.7.0 at some point. Someone must have fixed the tags because bumping to 7 was unwarranted.

Reply:
I bumped it to `0.7.0` when I added the new `get_unit_events()` method.

Reply:
Okay. Should the version and tag match?

Reply:
Ideally, yes. I'm fine with setting them both to `0.6.3`.

Reply:
👌